sdk/monitor/azure-opentelemetry-exporter-azuremonitor/src/main/java/com/azure/opentelemetry/exporter/azuremonitor/AzureMonitorExporter.java

```diff
@@ -406,6 +407,7 @@ private void exportRequest(String stdComponent, SpanData span, List<TelemetryIte
         }
 
         Double samplingPercentage = removeAiSamplingPercentage(attributes);
+        samplingPercentage = samplingPercentage == null ? 100.0 : samplingPercentage;
```

Comment:
Is/Should this be defaulted by the exporter when provided null?

Reply:
Sampling will be completely removed as it won't be supported in this exporter.